documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -32,6 +32,10 @@ node build
 
 Development dependencies will be bundled into your app using [Rollup](https://rollupjs.org). To control whether a given package is bundled or externalised, place it in `devDependencies` or `dependencies` respectively in your `package.json`.
 
+### Compressing responses
+
+You will typically want to compress responses coming from the server. You will often deploy your server behind a load balancer or reverse proxy. Since Node.js is single-threaded, it typically results in better performance to handle compression at that layer rather than directly in the server. If you build a [#custom-server](custom server) and do want to add a compression middleware there, note that we would recommend using `@polka/compression` since SvelteKit streams responses and the `compression` package does not support streaming and may error when used.
```

Comment:
```suggestion
You will typically want to compress responses coming from the server. If you are already deploying your server behind a load balancer or reverse proxy for SSL, it typically results in better performance to also handle compression at that layer since Node.js is single-threaded.

However, if you're building a [#custom-server](custom server) and do want to add a compression middleware there, note that we would recommend using [`@polka/compression`](https://www.npmjs.com/package/@polka/compression) since SvelteKit streams responses and the `compression` package does not support streaming and may cause errors when used.
```

Some suggested wording adjustments, mention why you might already want to be using a reverse proxy, and a link to the npm page for `@polka/compression`. The link is what I feel the most strongly about - feel free to adjust or disregard the other wording suggestions.

Reply:
I made a couple minor changes when merging your suggestion, so you may want to take another look